Sources/PT.PM.JavaParseTreeUst/Converter/StatementVisitor.cs

```diff
@@ -62,6 +63,23 @@ public UstNode VisitLocalVariableDeclaration(JavaParser.LocalVariableDeclaration
                 .Select(varDec => (AssignmentExpression)Visit(varDec))
                 .Where(initializer => initializer != null).ToArray();
 
+            if (initializers.Count() == 1 && initializers.First().Right is MultichildExpression multichildExpression)
```

Comment:
Simplify: 
```CSharp
if (initializers.SingleOrDefault()?.Right is MultichildExpression multichildExpression)
```

Reply:
SingeOrDefault throws an exception if there is more than one element in the sequence. It's unwanted behavior

Reply:
Ok.